src/keycloak/client-config.dto.ts

```diff
@@ -0,0 +1,3 @@
+export class ClientConfig {
```

Comment:
is this from Keycloak / generation during setup? A very brief JsDoc would be helpful :blush:

Reply:
These are extracts of the relevant configs that we use for creating clients and realms. I added a description.